components/tokens/custom-spectrum/custom-darkest-vars.css

```
@@ -17,6 +17,15 @@ governing permissions and limitations under the License.
   --spectrum-menu-item-background-color-down: var(--spectrum-transparent-white-200);
   --spectrum-menu-item-background-color-key-focus: var(--spectrum-transparent-white-200);
 
-    /* Drop Zone background color rgb */
-    --spectrum-drop-zone-background-color-rgb: var( --spectrum-blue-900-rgb); /* var(--spectrum-accent-color-900);*/
+  /* Drop Zone background color rgb */
+  --spectrum-drop-zone-background-color-rgb: var(--spectrum-blue-900-rgb); /* var(--spectrum-accent-color-900);*/
+
+  --spectrum-calendar-day-background-color-selected: rgba(64,150,243,0.2);
```

Are we able to use this instead?

```suggestion
  --spectrum-calendar-day-background-color-selected: rgba(var(--spectrum-blue-800-rgb), 0.2);
```